app/controllers/Config.scala

```diff
@@ -13,9 +13,10 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-package utils
+package controllers
 
-import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
-import spray.json._
+object Config {
 
-trait JsonSupport extends SprayJsonSupport with DefaultJsonProtocol
\ No newline at end of file
+  val server="https://delphi.cs.uni-paderborn.de/api-legacy/search?pretty"
```

Comment:
Please do not hard code URLs in the code... Especially production URLs...

Reply:
This also has been updated with env variables. Please verify. Thanks!

Reply:
This is resolved, however there are new comments on this issue below.